frontend/package.json

```diff
@@ -20,9 +20,9 @@
   ],
   "main": "index.ts",
   "scripts": {
-    "ng": "./node_modules/@angular/cli/bin/ng",
-    "tsc": "./node_modules/typescript/bin/tsc",
-    "i18n-extract-from-source": "./node_modules/@angular/cli/bin/ng extract-i18n --out-file ./src/locale/messages.xlf",
+    "ng": "node ./node_modules/@angular/cli/bin/ng",
+    "tsc": "node ./node_modules/typescript/bin/tsc",
+    "i18n-extract-from-source": "node ./node_modules/@angular/cli/bin/ng extract-i18n --out-file ./src/locale/messages.xlf",
```

Comment:
How is this related to adding Tor support to fetch static assets? Was this committed by mistake?